deployment-scanner/src/main/java/org/jboss/as/server/deployment/scanner/FileSystemDeploymentService.java

```diff
@@ -274,7 +274,7 @@ public void run() {
         assert scheduledExecutor != null;
         assert deploymentDir != null;
 
-        this.resourceAddress = resourceAddress.toModelNode();
+       this.resourceAddress = resourceAddress.toModelNode();
```

Comment:
The tests are failing so you'll need to update this branch to fix that, so might as well correct this whitespace. :)